src/ochanticipy/datasources/glofas/glofas.py

```diff
@@ -138,6 +139,15 @@ def __init__(
             dtstart=self._start_date,
             until=self._end_date,
         )
+        if self._date_range.count() > _CDS_MAX_REQUESTS:
```

Comment:
So, question, could an eventual implementation make the maximum requests and then sleep before making the additional ones, preventing the user from having to create two separate instances if they want to analyse data across a time that contains more points than `_CDS_MAX_REQUESTS`?

Reply:
Not saying to be done now, but should it be an issue or no?

Reply:
Oh yeah definitely, good question. It's actually how I started writing this module in the first place, using `asyncio` and a producer / consumer architecture. Then I realized that the request limit was very high and we wouldn't reach it (at the time, this was a few months ago...) so I dropped this plan, as `asyncio` makes things more complicated and testing hard. 

But _now_ with the forecast we do reach the queue limit if asking for all historical available forecasts, oops. So indeed this is a temporary fix.

Reply:
Ah yes  and it should be an issue, let me add it